packages/oid4vci-holder/src/agent/OID4VCIHolder.ts

```diff
@@ -215,6 +216,7 @@ export class OID4VCIHolder implements IAgentPlugin {
     oid4vciHolderStoreCredentialBranding: this.oid4vciHolderStoreCredentialBranding.bind(this),
     oid4vciHolderStoreCredentials: this.oid4vciHolderStoreCredentials.bind(this),
     oid4vciHolderSendNotification: this.oid4vciHolderSendNotification.bind(this),
+    oid4vciGetFederationTrust: this.oid4vciGetFederationTrust.bind(this),
```

Comment:
Was this supposed to go into oid4vci-holder?
Quote from Niels in my PR:
> OIDF has nothing to do with OID4VC, except for the fact it can be used as a registry for trust establishment. So it for sure does not belong in the VCI REST API or anything

Reply:
Yes should be integrated here. No the function should not be exposed to the outside world

Reply:
not exposed anymore